### PR TITLE
README: Document "Failed set host name" error

### LINKED ISSUE #363: Sometimes, foxbox needs to run as root.
Today, when I tried to run foxbox, (I tried with and without --disable-tls and also tried removing the certs directory, all with the same effect):

```
2080 >RUST_BACKTRACE=1 cargo run -- --disable-tls
     Running `target/debug/foxbox --disable-tls`
thread '<main>' panicked at 'Failed set host name: Access denied (code -20)', /home/dhylands/.multirust/toolchains/nightly-2016-04-10/cargo/git/checkouts/multicast-dns-d51af2fa146824d6/a6e4bcc/src/adapters/avahi/adapter.rs:346
stack backtrace:
   1:     0x7ff3d349fe80 - std::sys::backtrace::tracing::imp::write::h714760a4c8c0cdd8
   2:     0x7ff3d34a5fab - std::panicking::default_hook::_$u7b$$u7b$closure$u7d$$u7d$::hff309ab1d83ffd90
   3:     0x7ff3d34a5c33 - std::panicking::default_hook::h08ad3bb09872855b
   4:     0x7ff3d348e17f - std::sys_common::unwind::begin_unwind_inner::h406d5f1a330b854b
   5:     0x7ff3d348f1c8 - std::sys_common::unwind::begin_unwind_fmt::h57ea3fbee1a40196
   6:     0x7ff3d2d49f38 - _<adapters..avahi..adapter..AvahiAdapter as adapters..adapter..HostAdapter>::set_name::h2bed60aa1ab07b54
                        at /home/work/fxbox/foxbox/<std macros>:8
   7:     0x7ff3d2d4ec0f - multicast_dns::host::host_manager::HostManager::set_name::h8b6b157855e59ff3
                        at /home/dhylands/.multirust/toolchains/nightly-2016-04-10/cargo/git/checkouts/multicast-dns-d51af2fa146824d6/a6e4bcc/src/host/host_manager.rs:21
   8:     0x7ff3d2772fa6 - foxbox::update_hostname::hc21239e54baa1f82
                        at src/main.rs:167
   9:     0x7ff3d27737ce - foxbox::main::h000728ee4216cb83
                        at src/main.rs:241
  10:     0x7ff3d34a5874 - std::sys_common::unwind::try::try_fn::hfe4b14ce00a3f42f
  11:     0x7ff3d349f05b - __rust_try
  12:     0x7ff3d34a52bb - std::rt::lang_start::h12d9b3310b48a726
  13:     0x7ff3d279bed9 - main
  14:     0x7ff3d0a08ec4 - __libc_start_main
  15:     0x7ff3d276ecc6 - <unknown>
  16:                0x0 - <unknown>
error: Process didn't exit successfully: `target/debug/foxbox --disable-tls` (exit code: 101)
```

In order to get it to run, I had to run it as root. Then I was able to run it as non-root.

However, running it as means that any files produced were owned by root (so it created a new certs directory).

This feels problematic to me.

Tagging @samgiles since he added the code which calls update_hostname.

### PR DESCRIPTION
**Pull request by JohanLorenzo**
Fixes #363 

Follow up #281 cc @samgiles 

r? @azasypkin @dhylands

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,38 +93,19 @@ authentication = []
...
-Currently you would likely want to start the daemon like this:
+You will be disabling [TLS](https://wiki.mozilla.org/Connected_Devices/Projects/Project_Link/TLS) support. We hope to have out-of-the-box TLS support ready pretty soon, but for now disabling it is the easiest way to run foxbox.
 
-```bash
-cargo run -- -r http://knilxof.org:4242 --disable-tls
-```
+### More options
 
-That means that your foxbox will be using our dev [registration server](https://wiki.mozilla.org/Connected_Devices/Projects/Project_Link/Registration_Server) and you will be disabling [TLS](https://wiki.mozilla.org/Connected_Devices/Projects/Project_Link/TLS) support. We hope to have out-of-the-box TLS support ready pretty soon, but for now disabling it is the easiest way to run foxbox.
+To see every available options, run: `cargo run -- --help`. Here below are detailed some specific configurations.
```
Comment (JohanLorenzo) [inline README.md:106]: Nit: option~~s~~

**Comment by samgiles** [comment]
Nice, thanks for that.  r+

**Comment by azasypkin** [comment]
lgtm, thanks! r+

**Comment by JohanLorenzo** [comment]
There hasn't been any activity on this PR for more than 2 years. Although, I see some fxbox repos still have some activity. I'm closing this PR for the sake of cleanup. Please reopen if you think it still has some use.

**Pull request closed without merging**